### DISCUSSION TITLE
Ask For ASA Version

### DISCUSSION THREAD

**Discussion by vaascape** [coldino/Larkator#90] [Q&A]
Hey thank you for your brilliant work! Any chance to make a ASA version?

**Comment by coldino** [comment]
Sadly that's not really possible at the moment as there's a totally new savegame file format that we don't know how to read.

My focus at the moment is on extracting data for Ark Smart Breeding and the wiki, but other smart people are looking into the savegame format and I'll check in with them once there is significant progress.

**Reply by Magirb** [reply]
Nice to see you are responding to this! Any chance of this being updated to work with ASE again? All I get when I try to use it is Found Unknown Version 11.
